tests/http-request-invalid/test.yaml

```diff
@@ -0,0 +1,7 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
```

Comment:
we should set the min-version here, probably 4.1.0 as we'll backport a fix to the master-4.1.x branch in Suricata.

Reply:
Ok